cli/src/test/java/com/devonfw/tools/ide/commandlet/UninstallCommandletTest.java

```diff
@@ -84,6 +103,35 @@ public void testUninstallNpm() {
     assertThat(context).log().hasEntries(IdeLogEntry.ofSuccess("Successfully uninstalled npm"));
   }
 
+  /** Test of {@link UninstallCommandlet} run with --force. */
+  @ParameterizedTest
+  @ValueSource(strings = { "windows", "mac", "linux" })
+  public void testForceUninstallAzure(String os) {
+
+    // arrange
+    SystemInfo systemInfo = SystemInfoMock.of(os);
+    String tool = "az";
+    IdeTestContext context = newContext(PROJECT, null, true);
```

Comment:
BTW: If you set `copyForMutation` to `true` what is required and correct here, you can simplify to the standard call:
```suggestion
    IdeTestContext context = newContext(PROJECT);
```